ext/spring5/src/main/java/org/glassfish/jersey/server/spring/scope/package-info.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2013, 2019 Oracle and/or its affiliates. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0, which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception, which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ */
+
+/**
+ * Jersey server-side Spring 4 integration injection scopes related classes.
```

Comment:
Spring 5?

also it would be great to add @ since version here and in other package-info files. Coming version is 2.29

Actually all included package-info.java files have same reference to Spring 4. I suppose it shall be corrected to Spring 5.

Reply:
@senivam Both package-info.java corrected to Spring 5 and @ since 2.29 added.